Source/WebCore/platform/graphics/FontCascadeDescription.h

```diff
@@ -160,7 +160,7 @@ class FontCascadeDescription : public FontDescription {
 
 inline bool FontCascadeDescription::operator==(const FontCascadeDescription& other) const
 {
-    return FontDescription::operator==(other)
+    return static_cast<FontDescription>(*this) == static_cast<FontDescription>(other)
```

Comment:
How did you decide to cast to `FontDescription` rather than to `const FontDescription&`?

Reply:
That's my mistake, I'll fix.